src/server/trpc/router/stats.ts

```diff
@@ -12,8 +14,42 @@ export const statsRouter = router({
         description: true,
         isPublic: true,
         locationId: true,
-		assignmentId: true,
+		    assignmentId: true,
+      },
+    });
+  }),
+  getTicketStatsHelpedByUser: publicProcedure
+    .input(z.object({ 
+      userId: z.string() 

```

Comment:
Instead of passing in the current user's ID, you can access it through `ctx.session?.user?.id` and then not require an input